overdrive.py

```diff
@@ -344,12 +343,15 @@ def get(self, url, extra_headers, exception_on_401=False):
         else:
             return status_code, headers, content
 
+    @property
+    def token_authorization_header(self):
+        s = b"%s:%s" % (self.client_key, self.client_secret)
+        return "Basic " + base64.standard_b64encode(s).strip()
+
```

Comment:
`token_post` itself gets mocked by the `MockOverdriveAPI`. I moved this to a separate method so I could test it in isolation.